SRS/SRS_OC_BODBAW/srs_oc_bodbaw_netcdf_creation.py

```diff
@@ -18,6 +18,9 @@
 import pyexcel
 import sys
 import tempfile
+import warnings
+sys.path.insert(0, os.path.join(os.environ.get('DATA_SERVICES_DIR'), 'lib'))
+from python.ship_callsign import ship_callsign_list
```

Comment:
`DATA_SERVICES_DIR/lib/python` is already on `sys.path`, so you should be able to just say
`from ship_callsign import ship_callsign_list` without adding anything to it.